t/15-assets.t

```diff
@@ -182,16 +185,19 @@ ok($fixed->is_fixed(), 'fixed should be considered a fixed asset');
 # test Utils::locate_asset
 # fixed HDD asset
 my $expected = catfile($OpenQA::Utils::assetdir, 'hdd', 'fixed', 'fixed.img');
-is(locate_asset('hdd', 'fixed.img', 1), $expected, 'locate_asset should find fixed asset in fixed location');
+is(locate_asset('hdd', 'fixed.img', mustexist => 1), $expected, 'locate_asset should find fixed asset in fixed location');
+# relative
+$expected = catfile('hdd', 'fixed', 'fixed.img');
+is(locate_asset('hdd', 'fixed.img', mustexist => 1, relative => 1), $expected, 'locate_asset should return fixed path as relative');
 
 # transient repo asset
-$expected = catfile($OpenQA::Utils::assetdir, 'repo', 'testrepo');
-is(locate_asset('repo', 'testrepo', 1), $expected, 'locate_asset should find testrepo in expected location');
+$expected = catfile($OpenQA::Utils::assetdir, 'repo', 'tmprepo');
+is(locate_asset('repo', 'tmprepo', mustexist => 1), $expected, 'locate_asset should find tmprepo in expected location');
 
 # non-existent ISO asset
 $expected = catfile($OpenQA::Utils::assetdir, 'iso', 'nex.iso');
-is(locate_asset('iso', 'nex.iso', 0), $expected, 'locate_asset 0 should give location for non-existent asset');
-ok(!locate_asset('iso', 'nex.iso', 1), 'locate_asset 1 should not give location for non-existent asset');
+is(locate_asset('iso', 'nex.iso'), $expected, 'locate_asset 0 should give location for non-existent asset');
```

Comment:
well, this was intended to be a test of what happens when you explicitly set `0`, not a test of 'whatever the default currently is'. Of course to be truly comprehensive we would test both, I guess.